mvp-spec/json-schemas/transmission/transmission-response.json

```diff
@@ -0,0 +1,42 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "title": "Transmission Response",
+    "description": "",
+    "$id": "transmission-response",
+    "type": "object",
+    "properties": {
+        "version": {
+            "$ref": "version.json"
+        },
+        "transaction_id": {
+            "$ref": "transaction_id.json"
+        },
+        "receiver": {
+            "$ref": "transmission-receiver.json"
```

Comment:
We want a dedicated "description" and clear "example".

Reply:
you can "override" both here if needed.
Or, you could instead use a reference to `../domain.json` in `transmission-receiver.json`.
I think it can be valuable to centralize how we define a domain in the whole framework, don't you think? (not a major thing, but I think it could be good, just as we did for `version`, for example)

Reply:
ok